src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-eks-add-on.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: New Relic Kubernetes Operator from AWS Marketplace
+tags:
+  - Infrastructure
+  - Install the EKS Add-on
+  - NewRelic Operator
+  - AWS EKS Add-on
+metaDescription: New Relic's EKS Add For EKS Monitoring.
+---
+
+import eksAddOnTab from 'images/EKS-Add-On-Screenshot-1.webp'
+
+import eksAddOnMarketplace from 'images/EKS-Add-On-Screenshot-2.webp'
+
+import eksAddOnMarketplaceConfig from 'images/EKS-Add-On-Screenshot-3.webp'
+
+<CollapserGroup>
+  <Collapser
+    id="what-is-extension"
+    title="What is an AWS EKS?"
+  >
+    Amazon Elastic Kubernetes Service (Amazon EKS) is a managed Kubernetes service that makes it easy for you to run Kubernetes on AWS and on-premises. Kubernetes is an open-source system for automating deployment, scaling, and management of containerized applications.
+  </Collapser>
+</CollapserGroup>
+
+## This document shows the steps required to use EKS AddOn from AWS Marketplace.
+#### **NOTE : This addon is currently supported only on X86 instance types and the support for ARM64 architecture type is coming soon**
+
+## Pre-Requisites
+1. [Amazon Elastic Kubernetes Service (EKS)](https://docs.aws.amazon.com/eks/latest/userguide/what-is-eks.html) Cluster. See the userguide here
+2. A valid New Relic. You can subscribe to New Relic from AWS Marketplace with Free Tier. See details [here](https://aws.amazon.com/marketplace/pp/prodview-ov56chowabeb4?sr=0-3&ref_=beagle&applicationId=AWS-Marketplace-Console)
+3. Subscribe to [New Relic Kubernetes Integration](https://aws.amazon.com/marketplace/pp/prodview-gcywa6keq2ajy?applicationId=AWS-Marketplace-Console&ref_=beagle&sr=0-5), available for free
+4. Login to [New Relic](https://one.newrelic.com/) and generate a [License Key](https://one.newrelic.com/api-keys)
+
+## Provision New Relic Kubernetes Operator EKS AddOn via AWS console
+1. Navigate to the EKS Cluster > Add-Ons > Get More add-ons
+    <img
+      title="eksAddOnTab.png"
+      alt="EKS Add On Tab."
+      src={eksAddOnTab}
+    />
+2. Under AWS Marketplace add-ons, search, browse or navigate to New Relic Kubernetes Integration , check mark against the result and click Next
+    <img
+      title="eksAddOnMarketplace.png"
+      alt="EKS Add On Marketplace."
+      src={eksAddOnMarketplace}
+    />
+3. Select the latest default version or any pinned down version of your interest and click Next
```

Comment:
"pinned down version" is not a common phrase. Instead, say "or choose an alternate version, if required, and click Next"